app/helpers/spotlight/pages_helper.rb

```diff
@@ -37,11 +37,11 @@ def nestable_data_attributes(type)
     def nestable_data_attributes_hash(type)
       case type
       when 'feature_pages'
-        { "data-max-depth": '2',
-          "data-expand-btn-HTML": '',
-          "data-collapse-btn-HTML": '' }
+        { "data-max-depth" => '2',
+          "data-expand-btn-HTML" => '',
+          "data-collapse-btn-HTML" => '' }
       when 'about_pages'
-        { "data-max-depth": '1' }
+        { "data-max-depth" => '1' }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.